plugins/legacy-preset-chart-nvd3/src/utils/tokenize.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { validateNumber } from '@superset-ui/validator';
+
+const tokenizeToNumericArray = value => {
```

Comment:
* Perhaps a classic function and inline export is more readable here.
* (optional) This file can also be written in typescript 

```ts
export function tokenizeToNumericArray(value) {
}
```

Reply:
Will do 👍